modules/dashboard/php/NDB_Form_dashboard.class.inc

```diff
@@ -36,7 +36,7 @@ class NDB_Form_Dashboard extends NDB_Form
         $DB     = Database::singleton();
         $user   = User::singleton();
         $config = NDB_Config::singleton();
-        $site   = $user->getSiteName();
+        $site   = explode(';', $user->getSiteName());
```

Comment:
In the php/libraries/User.class.inc:
I changed the getSiteName() to return array, but it should be left as string as the basic query in the same file is group concatenating the sites (only getCenterID should be changed to return array). So I corrected this.